### ISSUE TITLE
no file /train/0.png

### ISSUE THREAD

**Issue by gydlcc** [Bartzi/stn-ocr#20]
hello dear Bartzi
FileNotFoundError: [Errno 2] No such file or directory: '/train/0.png'
what i'm supposed to do with that

**Comment by Bartzi** [comment]
You should have a look at your groundtruth file. The path to your images is definitely wrong. I think you should not have `/train/0.png` but rather `train/0.png`

**Comment by gydlcc** [comment]
yeah, i guess so . but how could i change    /train/0.png into   train/0.png .  i don't know where to change

**Comment by gydlcc** [comment]
what i am running is train_svhn.py

**Comment by Bartzi** [comment]
You already figured out which `csv` file to use for training. This csv file provides the path to all images that are to be used for training. You should have a look at this file.
